Source/WTF/wtf/text/StringView.cpp

```diff
@@ -535,6 +535,21 @@ void StringView::setUnderlyingString(const StringView& otherString)
     adoptUnderlyingString(underlyingString);
 }
 
-#endif // CHECK_STRINGVIEW_LIFETIME
+#else
+
+bool StringView::underlyingStringIsValidImpl() const
```

Comment:
> I didn't realize that the plan involved uninlining these functions in release builds. That's a performance issue.

These functions aren't actually being un-inlined when building WebKit in release - if you check the header, this class will still use the same implementations of `underlyingStringIsValid()` and `setUnderlyingString()`, which are still no-ops and can be trivially inlined/eliminated. _These_ no-op functions are only exported (in release builds) for the sake of apps that build in Debug against a Release WebKit. 

> Would it work to just add an export qualifier to these inlines functions, so that they are exported? I think that it should

I was concerned about the possibility that exporting these no-ops might impact the compiler/linker's ability or willingness to inline them, so I chose to leave the inlines as-is in Release out of maximum paranoia.

Reply:
Yes, I misread your patch. Seems like a do that a lot before 9 am :)